backend/middlewares/errorMiddleware.ts

```diff
@@ -0,0 +1,21 @@
+import { Request, Response, NextFunction } from 'express';
+import { Prisma } from '@prisma/client';
+
+const handleError = async (err: Prisma.PrismaClientKnownRequestError | string, req: Request, res: Response, next: NextFunction) => {
+
+    if (typeof (err) === 'string') return res.status(401).json(err);
+
+    let responseMsg;
+    switch (err.code) {
+        case 'P2002':
```

Comment:
Why is this empty?

Reply:
This case was not empty, both of the cases were doing the same thing. However the code updated to avoid responding with direct DB messages.